tests/e2e/suites/authn-basic.lc.neg.test.ts

```diff
@@ -9,27 +9,25 @@
  */
 
 import { setupAndGo } from '../utilities/setup-and-go';
+import browsers from '../utilities/browsers';
 
 describe('Test bad login flow', () => {
-  ['chromium', 'webkit', 'firefox'].forEach((browserType) => {
+  browsers.forEach((browserType) => {
     it(`Login UNsuccessfully with ${browserType}`, async (done) => {
-      const { browser, page } = await setupAndGo(browserType, 'authn-basic/', {
-        pw: 'wrong_password_123!',
-      });
+      try {
+        const { browser, messageArray } = await setupAndGo(browserType, 'authn-basic/', {
+          pw: 'wrong_password_123!',
+          selector: '.Auth_Error',
+        });
 
-      const messageArray = [];
+        // Test assertions
+        expect(messageArray.includes('Error: Auth_Error')).toBe(true);
 
-      page.on('console', (msg) => {
```

Comment:
Event listeners and waits for selectors are now all contained in the `setupAndGo` function.